python/pyspark/sql/streaming/listener.py

```diff
@@ -197,14 +220,24 @@ class QueryProgressEvent:
     Event representing any progress updates in a query.
 
     .. versionadded:: 3.4.0
+    .. versionchanged:: 3.5.0
+        Add fromJson constructor to support Spark Connect.
```

Comment:
I think it's fine to remove this. The constructor was already supposed to be private.